README.md

```diff
@@ -119,6 +119,62 @@ ClientConfig = [{reconnect_cool_down_seconds, 10}],
 ok = brod:start_client([{"localhost", 9092}], brod_client_1, ClientConfig).
 ```
 
+## Handling of connection errors
+
+Usually you would have a supervisor that starts your worker module. In the worker's `start_link` method, you should do two things:
+
+1. Make sure the Brod client you're going to use is running by calling `brod:start_client`. That will start the client under Brod's supervisor, if a client with the given name does not already exist. Afterwards, Brod will automatically recover from connection issues.
+1. Start, e.g., a group subscriber by calling `brod:start_link_group_subscriber`, and return the pid for your supervisor to monitor.
+
+(Note that using `brod:start_link_client` to link the client to your supervisor is deprecated. Using it, a connection loss won't trigger a reconnect, even though the process linked to stays alive.)
```

Comment:
You are right.

> Using it, a connection loss won't trigger a reconnect.

This made me thought that you meant ANY connection loss won't trigger a reconnect.
But the fact is `brod_client` only crashes when ALL of the cluster members are unreachable.

>  Even though the process linked to stays alive

I don't think it's necessary to state the obvious because
ANY `start_link` caller should expect the returned pid to exit.
So it should either trap exit to receive an 'EXIT' message or
unlink then monitor it to receive a 'DOWN' message, and finally
apply whatever restart strategy fits the best to their use case.